xwalk_android.gypi

```diff
@@ -213,6 +213,58 @@
       'includes': ['../build/jni_generator.gypi'],
     },
     {
+      'target_name': 'xwalk_runtime_lib_lzma_compress',
+      'type': 'none',
+      'dependencies': [
+        'xwalk_runtime_lib_apk',
+      ],
+      'variables': {
+        'files_to_compress': [
+          '<(PRODUCT_DIR)/xwalk_runtime_lib_apk/stripped_libraries/libxwalkcore.so',
+          '<(PRODUCT_DIR)/xwalk_runtime_lib_apk/classes.dex',
+          '<(PRODUCT_DIR)/xwalk_runtime_lib/assets/icudtl.dat',
+          '<(PRODUCT_DIR)/xwalk_runtime_lib/assets/xwalk.pak',
+        ],
+        'assets_dir': '<(PRODUCT_DIR)/xwalk_runtime_lib_lzma/assets/',
+        'stamp': '<(PRODUCT_DIR)/xwalk_runtime_lib_lzma/gen.stamp',
+      },
+      'actions': [
+        {
+          'action_name': 'lzma_compress',
+          'message': 'compress ingredients for runtime APK',
+          'inputs': [
```

Comment:
You need `<@(files_to_compress)` here, otherwise you'll only run this script when the script itself has changed.